src/GraphQL/Execution/ExecutionStrategy.cs

```diff
@@ -203,6 +203,24 @@ protected static ExecutionNode BuildExecutionNode(ExecutionNode parent, IGraphTy
             };
         }
 
+        /// <summary>
+        /// Builds an execution node with the specified parameters.
+        /// </summary>
+        public static ExecutionNode BuildExecutionNode(ExecutionNode parent, IGraphType graphType, Field field, FieldType fieldDefinition, int? indexInParentNode, object source)
```

Comment:
Let's rename it to `BuildSubscriptionExecutionNode`, make `protected` and move into `SubscriptionExecutionStrategy`.

Reply:
ok